src/components/Aframe/my-loader.js

```diff
@@ -298,7 +296,7 @@ AFRAME.registerComponent("myloader", {
     }
   },
 
-  loadModel: function () {
+  loadModel: function (volueDataObject) {
```

Comment:
typo in variable name (`volue`, maybe instead of `Object` call it `Options`

Reply:
fixed